src/SDKs/SqlManagement/Sql.Tests/AzureRmSqlServerActiveDirectoryAdministratorTest.cs

```diff
@@ -22,30 +17,14 @@ public void TestSetServerActiveDirectoryAdministrator()
         {
             string testPrefix = "sqlcrudtest-";
             string suiteName = this.GetType().FullName;
-            string login = "dummylogin";
-            string password = "Un53cuRE!";
-            string version12 = "12.0";
             string aadAdmin = "DSEngAll";
 
             Dictionary<string, string> tags = new Dictionary<string, string>();
-            SqlManagementTestUtilities.RunTestInNewResourceGroup(suiteName, "TestSetServerActiveDirectoryAdministrator", testPrefix, (resClient, sqlClient, resourceGroup) =>
+            SqlManagementTestUtilities.RunTestInNewV12Server(suiteName, "TestSetServerActiveDirectoryAdministrator", testPrefix, (resClient, sqlClient, resourceGroup, server) =>
             {
-                // Create a server
-                string serverName = SqlManagementTestUtilities.GenerateName(testPrefix);
-                var server = sqlClient.Servers.CreateOrUpdate(resourceGroup.Name, serverName, new Microsoft.Azure.Management.Sql.Models.Server()
-                {
-                    AdministratorLogin = login,
-                    AdministratorLoginPassword = password,
-                    Version = version12,
-                    Location = SqlManagementTestUtilities.DefaultLocation,
-                    Tags = tags,
-                });
-
-                SqlManagementTestUtilities.ValidateServer(server, server.Name, login, version12, tags, SqlManagementTestUtilities.DefaultLocation);
-
                 // Add new Active Directory Admin
                 ServerAzureADAdministrator newAdmin = new ServerAzureADAdministrator("DSEngAll", new Guid("5e90ef3b-9b42-4777-819b-25c36961ea4d"), new Guid("72f988bf-86f1-41af-91ab-2d7cd011db47"), "sqlcrudtest-3421", "2c647056-bab2-4175-b172-493ff049eb29");
-                ServerAzureADAdministrator createResult = sqlClient.ServerAzureADAdministrators.CreateOrUpdateAzureADAdministrator(resourceGroup.Name, server.Name, newAdmin);
+                ServerAzureADAdministrator createResult = sqlClient.ServerAzureADAdministrators.CreateOrUpdate(resourceGroup.Name, server.Name, newAdmin);
```

Comment:
Good point, since we didn't get this feedback on the original PR (https://github.com/Azure/azure-sdk-for-net/pull/3196) can we fix this as a separate issue?